NEWS

```diff
@@ -149,6 +149,13 @@ CHANGES WITH 244 in spe:
 
         * libcryptsetup >= 2.0.1 is now required.
 
+	* systemd-resolved validates IP addresses in certificates now when GnuTLS
+	  is being used.
+
+	* systemd-resolved prefers TLS 1.3 before TLS 1.2 now.
+
+	* machinectl reboot actually works now.
```

Comment:
We generally don't mention bug fixes, and we don't mention routine upgrades of versions. I'd certainly remove the last item, and possible the second one too.

Reply:
totally fine for me.

Reply:
I have removed the last two items and force pushed again.